Bottom: 6a212b0273e03cc43c7e2fe9da6c533e630113c9
Top:    b12aeb1b086d26f1d138861e16db8f739988421b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-09 17:07:06 -0500

Add parameter for buildbot version



---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 521c823..dde9cdb 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -1,4 +1,5 @@
-FROM        buildbot/buildbot-master:v0.9.13
+ARG         BUILDBOT_VERSION=latest
+FROM        buildbot/buildbot-master:$BUILDBOT_VERSION
 MAINTAINER  Andrew Cooper <andrew.cooper@novatechweb.com>
 
 WORKDIR /buildbot
